src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -60,18 +69,42 @@ export default function ConfirmedAppointmentDetailsPage() {
 
   useEffect(
     () => {
-      const pageTitle = isCommunityCare ? 'Community care' : 'VA';
+      let pageTitle;
+      let prefix = null;
+
+      if (isPast) prefix = 'Past';
+      else if (isCanceled) prefix = 'Canceled';
+
+      if (isCommunityCare)
+        pageTitle = prefix ? `${prefix} community care` : 'Community care';
+      else if (isInPerson) {
+        if (appointment?.vaos?.isCompAndPenAppointment)
+          pageTitle = prefix ? `${prefix} claim exam` : 'Claim exam';
+        else pageTitle = prefix ? `${prefix} in-person` : 'In-person';
+      } else if (isVideo) pageTitle = prefix ? `${prefix} video` : 'Video';
+      else if (isVAPhoneAppointment) {
+        pageTitle = prefix ? `${prefix} phone` : 'Phone';
+      }
+
       const pageTitleSuffix = featureBreadcrumbUrlUpdate
         ? ' | Veterans Affairs'
         : '';
       if (appointment && appointmentDate) {
         document.title = `${pageTitle} appointment on ${appointmentDate.format(
-          'dddd, MMMM D, YYYY',
+          'dddd, MMMM DD, YYYY',
```

Comment:
Hmm can we double check with design that this is what we actually want? I know in the specs it says `DD` but personally I find "August 09" to be odd. I like "August 9" better myself but just want to make sure we're all on the same page here.

cc @outerpress

Reply:
Checked with Peter and he agrees. No leading zero.

"No preceding zero pls. I think in this context it's easier to read without it"

cc @outerpress